### PR TITLE
Joeys branch

### PR DESCRIPTION
**Pull request by goughjo02**

### CODE REVIEW

**Review by antoniocassano84 — COMMENTED** [review]
is this change necessary?

File: README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
+
 Each new term in the Fibonacci sequence is generated by adding the previous two terms. By starting with 1 and 2, the first 10 terms will be:
 
 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, ...
 
 By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms.
 
-Available at: https://projecteuler.net/problem=2
-
+Available at: [Project Eurler](https://projecteuler.net/problem=2)
```
Comment (antoniocassano84) [inline README.md:9]: why this is necessary?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
+# Even Fibonacci Numbers
```
Comment (goughjo02) [inline README.md:1]: you should always have a heading at the top of a README

File: README.md (hunk 1 above)
Comment (goughjo02) [inline README.md:9]: you should not use bare links, you should use hyper links

Comment (goughjo02) [reply README.md:9]: it is a markdown standard to not have bar e links. You should use hyperlinks. It is just convention

Comment (antoniocassano84) [reply README.md:9]: fucking hell.. that stuff is boss... ta la!

**Review by antoniocassano84 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
+
```
Comment (antoniocassano84) [inline README.md:2]: # Even Fibonacci numbers

**Review by antoniocassano84 — APPROVED** [review]

**Pull request merged**